README.md

```diff
@@ -355,14 +414,18 @@ Nuxeo.oauth2.fetchAccessTokenFromJWTToken('http://localhost:8080/nuxeo', 'my-app
 });
 ```
 
-__Nuxeo.oauth2.refreshAccessToken(baseURL, clientId, refreshToken[, params])__
+__Nuxeo.oauth2.refreshAccessToken(optsbaseURL, clientId, refreshToken[, params])__
 
-Manually refresh an access token for the given `baseURL`, `clientId` and `refreshToken`.
+Manually refresh an access token for the given `opts` object containing `baseURL`, `clientId`, `refreshToken` and optional `clientSecret` and `params`.
```

Comment:
```suggestion
Manually refresh an access token for the given `opts` object containing `baseURL`, `clientId`, `refreshToken`, and optional `clientSecret` and `params`.
```

Reply:
baseURL`, `clientId`, `refreshToken`, optional `clientSecret`, (and) `params`